entropylab_qpudb/_qpudb_basedefs.py

```diff
@@ -0,0 +1,51 @@
+import dataclasses
+from dataclasses import dataclass
+from datetime import datetime
+from enum import IntEnum, auto
+from typing import Any
+
+from persistent import Persistent
+
+
+class CalState(IntEnum):
```

Comment:
Moving classes and types outside the qpudatabase.py file is not clear.
the name of this file is also unclear and doesn't imply what it should contain